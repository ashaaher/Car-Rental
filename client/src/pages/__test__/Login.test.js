import React from 'react';
import { shallow } from 'enzyme';
import Login from '../Login';

jest.mock('@okta/okta-react', () => ({
  useOktaAuth: () => {
  return {
    authState: {},
    authService: {}
  };
}
}));

it('Login Page matches snapshot', () => {
  const wrapper = shallow(<Login />);
  expect(wrapper).toMatchSnapshot();
})