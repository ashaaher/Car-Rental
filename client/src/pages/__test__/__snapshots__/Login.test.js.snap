// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page matches snapshot 1`] = `ShallowWrapper {}`;
